etc/kubernetes/manifests/overlays/local/observability/crds/customresourcedefinition-subscriptions.operators.coreos.com.yaml

```diff
@@ -0,0 +1,2636 @@
+﻿kind: CustomResourceDefinition
+apiVersion: apiextensions.k8s.io/v1
+metadata:
+  name: subscriptions.operators.coreos.com
+  uid: 2442059f-ce23-4d68-8f6d-1c9a6d1c0b6b
+  resourceVersion: '1947'
+  generation: 1
+  creationTimestamp: '2022-05-30T13:00:00Z'
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.6.2
+    include.release.openshift.io/ibm-cloud-managed: 'true'
+    include.release.openshift.io/self-managed-high-availability: 'true'
+  ownerReferences:
```

Comment:
looks like this is an export of a CR, is it correct ?

Reply:
Yes it was exported. Cleaned.